creating_S3_backend/guide.md

```diff
@@ -0,0 +1,42 @@
+# Creating an S3 Backend    
+
+## Introduction
+During the exercises in this registry you will need to have an S3 Backend configured. If you already have one you can bypass this but forn those who do not have one, this will be an in depth guide on how to make an S3 Backend using the [nozaq](https://registry.terraform.io/namespaces/nozaq) / [remote-state-s3-backend](https://registry.terraform.io/modules/nozaq/remote-state-s3-backend/aws/latest) module.
+
+## Prequisites
+In order to create an S3 Backend you will need the following software:
+- [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+
+And you will need the following accounts:
+- AWS
+
+## Step 0.1 - Navigation
```

Comment:
Dont need this, users are expected to have basic proficiency. This module is to make the s3 backend process easier - not teach tf